test/System/MemoryInputStream.cpp

```diff
@@ -0,0 +1,31 @@
+#include <SFML/System/MemoryInputStream.hpp>
+#include "SystemUtil.hpp"
+
+#include <doctest.h>
+
+TEST_CASE("sf::MemoryInputStream class - [system]")
+{
+    SUBCASE("Empty stream")
+    {
+        sf::MemoryInputStream mis;
+
+        CHECK(mis.read(nullptr, 0) == -1);
+        CHECK(mis.seek(0) == -1);
+        CHECK(mis.tell() == -1);
+        CHECK(mis.getSize() == -1);
+    }
+
+    SUBCASE("Temporary memory stream")
```

Comment:
I am not sure "Temporary" is meaningful here.

Reply:
I took the case name from the `FileInputStream` one, could just change it to something like "Memory stream open" or something

Reply:
In the context of `FileInputStream`, a temporary file is meaningful, but a temporary memory doesn't make so much sense. I think "Open memory stream" would be more meaningful here.